### PR TITLE
Fingerprint

### PR DESCRIPTION
**Pull request by AlexLartsev19**
- Add build fingerprint from FlymeOS 5.1.10.0A
- Clean up system.prop
- Add Doze package

### CODE REVIEW

File: system.prop [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #
+# from device/meizu/meizu6753_65cn_l1/system.prop
 #
```
Comment (Moyster) [inline system.prop]: It's not really needed as it's commented (#) and is just there for indication, revert this please :)

File: cm.mk [hunk 2]
```diff
@@ -28,3 +28,7 @@ PRODUCT_NAME := cm_m2note
 PRODUCT_BRAND := MEIZU
 PRODUCT_MODEL := M2 Note
 PRODUCT_MANUFACTURER := MEIZU
+
+PRODUCT_BUILD_PROP_OVERRIDES += \
```
Comment (Moyster) [inline cm.mk]: I just want to know why ^^, does it help ? Also, make it a separate pull request from MeizuDoze please :smile:

Comment (AlexLartsev19) [inline cm.mk]: Hm. You can fetch my repo with branch cm-12.1 and use git cherry-pick.Now i'm not at home and can't revert\make separate pull request, sorry :(

**Comment by Moyster** [comment]
don't worry nothing hurry ;) did you test the MeizuDoze change ?

**Comment by AlexLartsev19** [comment]
Hm, nope, but on Op3 and Yu it's works good :sun_with_face:

**Comment by Moyster** [comment]
Btw can you translate this in russian please ? 
https://github.com/Moyster/android_device_meizu_m2note/blob/cm-12.1/cmactions/res/values/arrays.xml

From line 4 to 14, answer here if you can't make a pull request, else the arrays.xml translated should go in cmactions/res/values-ru/

**Comment by Moyster** [comment]
And if you can check this : 
https://github.com/Moyster/android_device_meizu_m2note/blob/cm-12.1/cmactions/res/values-ru/strings.xml

Line 21 needs translation
Line 32 & 34 should say swipe left / swipe right
Line 42 should say : draw a vertical line down

Thanks in advance :)

**Comment by AlexLartsev19** [comment]
See this https://github.com/Moyster/android_device_meizu_m2note/pull/19

**Comment by AlexLartsev19** [comment]
@moyster, and what about MarshMallow/NouGat release plans? :)

**Comment by Moyster** [comment]
I'll close this one as I don't want to fake the build fingerprint (not necessary)
and the Doze package is in another PR.
